Applications/IoTBlock-test/main.cpp

```diff
@@ -13,15 +13,14 @@ using namespace vitroio::sdk;
 #define SERIAL_BAUDRATE 115200
 
 /**
- * Watchdog
+ * Watchdog ticker
```

Comment:
Do you mean `ticker` or `kicker`?

Reply:
@Silentsky0 in this case it is ticker which is used to clock watchdog kicker

Reply:
Alright, resolving